app/controllers/api/v1/configs_controller.rb

```diff
@@ -0,0 +1,103 @@
+class Api::V1::ConfigsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  rescue_from StandardError, with: :handle_standard_error
+  rescue_from ActiveRecord::RecordInvalid, with: :record_invalid
+  rescue_from ActiveRecord::RecordNotUnique, with: :record_not_unique
+  rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+  rescue_from ActionDispatch::Http::Parameters::ParseError, with: :parse_error
+
+  # GET api/v1/configs/name
+  def show
+    config = Config.friendly.find(params[:name])
+
+    render json: config.body, status: :ok
+  end
+
+  # POST api/v1/configs
```

Comment:
```suggestion
  # POST api/v1/configs/name
```
通常のアプリケーションとちがい、ID相当のnameをユーザーが指定するので、Createするときもパスでnameを指定する形にすると、UPDATEなどと対象的になって良いと思います。

おそらくルーティングのresoucesでは設定できなくて、専用のルーティングを指定する必要がありそうです。

Reply:
こちらで対応しました。
https://github.com/pubannotation/textae-configs/pull/13/commits/4bbf83ff367d70744e87c35f35972b5034725965